rho/utilities.py

```diff
@@ -486,8 +486,15 @@ def write_csv_data(keys, data, path):
         writer = csv.DictWriter(
             data_temp, fieldnames=sorted(keys), delimiter=',')
 
-        # Write a CSV header if necessary
-        writer.writeheader()
+        # Write a CSV header if necessary;
+        # handle Python 2.6 not having writeheader method
+        if sys.version_info[1] > 6:
```

Comment:
I think you should also look for the major version. Python 3.5 for example will not match and go to the else clause.